### ISSUE TITLE
[Artist Bio] Include partner that submitted bio

### ISSUE THREAD

**Issue by alloy** [artsy/emission#43]
As per https://github.com/artsy/eigen/issues/1133

* Include the name of the partner that submitted the bio.
* Don’t show partner submitted bios that have not been reviewed.

**Comment by katarinabatina** [comment]
hold for now, until we get confirmation on next steps from Genome Team

**Comment by sarahscott** [comment]
@katarinabatina what's the status on this one?

**Comment by katarinabatina** [comment]
I believe we can move forward, cc'ing @briansw if anything has changed.

**Comment by briansw** [comment]
Yes, we can show any bio that has been approved. We shouldn't show any that are awaiting review or dismissed.
